packages/iris-grid/src/AdvancedFilterCreator.tsx

```diff
@@ -592,19 +588,17 @@ class AdvancedFilterCreator extends PureComponent<
           {isValuesTableAvailable && !valuesTableError && (
             <>
               {!isBoolean && <hr />}
-              {valuesTable && (
-                <div className="form-group">
-                  <AdvancedFilterCreatorSelectValue<unknown>
-                    table={valuesTable}
-                    onChange={this.handleSelectValueChange}
-                    invertSelection={invertSelection}
-                    selectedValues={selectedValues}
-                    formatter={formatter}
-                    showSearch={!isDateType}
-                    timeZone={formatter.timeZone}
-                  />
-                </div>
-              )}
+              <div className="form-group">
+                <AdvancedFilterCreatorSelectValue<unknown>
+                  table={valuesTable}
```

Comment:
This should be giving you an error - you're passing in `valuesTable`, which could be `Table | undefined`... in `AdvancedFilterCreatorSelectValue` and `AdvancedFilterCreatorSelectValueList` table is a required prop. 

The problem is in `AdvancedFilterCreatorSelectValue` and `AdvancedFilterCreatorSelectValueList`, where we're assigning a defaultProp of `table: null`. I think what you _want_ is for those props to be optional (`table?: Table`), don't define the defaultProp, then leave this like it is here.